src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210427235819_AddTasksAndTaskPropertiesTables.cs

```diff
@@ -54,6 +55,114 @@ private void CreateTaskPropertiesTable(MigrationBuilder migrationBuilder)
                {
                    table.PrimaryKey($"PK_{DbTaskProperty.TableName}", x => x.Id);
                });
+
+            migrationBuilder.InsertData(
```

Comment:
I thought we can not change existing migrations, only add a new. Why you did not write a new migration?

Reply:
fixed